app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Response
```

Comment:
Please add tests for this module: `spec/controllers/concerns/response_spec.rb`

Reply:
Added ^)